src/controller/product-controller.js

```diff
@@ -0,0 +1,97 @@
+export class ProductController {
+  constructor(productService) {
+    this.productService = productService;
+  }

```

Comment:
```suggestion
  constructor(productService, likeService) {
    this.productService = productService;
    this.likeService = likeService;
  }
```
아래서 this.likeService를 사용하고 있으므로 DI 추가해줘야겠습니다.